startup_scripts/000_users.py

```diff
@@ -9,13 +9,17 @@
     sys.exit()
 
 for username, user_details in users.items():
-    if not User.objects.filter(username=username):
-        user = User.objects.create_user(
-            username=username,
-            password=user_details.get("password", 0) or User.objects.make_random_password(),
-        )
 
-        print("👤 Created user", username)
+    api_token = user_details.pop("api_token", Token.generate_key())
```

Comment:
Please use `None` or `0` instead of `Token.generate_key()` to change the behaviour back to what it was before.
We could create users without a token but now the  user will always get a token.

Reply:
OK. My assumption was that it would be good to just have the token generated because that would be the majority of use cases. And there was no means of generating that token automatically. Generating and entering the tokens to the YAML file is quite cumbersome.

If a user explicitly doesn't want a token generate then in the yaml file he can add an empty key:

```yaml
jdoe:
  api_token: 
```

Reply:
Ok, as long as it is possible to create a user without token, it is ok.